src/test/java/co/rsk/bitcoinj/script/StandardRedeemScriptParserTest.java

```diff
@@ -10,156 +10,116 @@
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
+import java.util.stream.Collectors;
 import org.junit.Assert;
 import org.junit.Before;
 import org.junit.Test;
 
 public class StandardRedeemScriptParserTest {
 
-    private final List<BtcECKey> btcECKeyList = new ArrayList<>();
-    private final BtcECKey ecKey1 = BtcECKey.fromPrivate(BigInteger.valueOf(100));
-    private final BtcECKey ecKey2 = BtcECKey.fromPrivate(BigInteger.valueOf(200));
-    private final BtcECKey ecKey3 = BtcECKey.fromPrivate(BigInteger.valueOf(300));
-    private final NetworkParameters networkParameters = NetworkParameters.fromID(NetworkParameters.ID_REGTEST);
+    private static final long CSV_VALUE = 52_560L;
+
+    private List<BtcECKey> defaultRedeemScriptKeys;
+    private StandardRedeemScriptParser standardRedeemScriptParser;
 
     @Before
     public void setUp() {
-        btcECKeyList.add(ecKey1);
-        btcECKeyList.add(ecKey2);
-        btcECKeyList.add(ecKey3);
+        defaultRedeemScriptKeys = RedeemScriptUtils.getDefaultRedeemScriptKeys();
```

Comment:
i would just call them "keys" for the standard redeem script parser

Reply:
Done